test/ssltest_old.c

```diff
@@ -1464,7 +1557,7 @@ int main(int argc, char *argv[])
     SSL_CTX_set_security_level(s_ctx, 0);
     SSL_CTX_set_security_level(s_ctx2, 0);
 
-    if (no_ticket) {
+    if (s_ticket1 == -1 && no_ticket) {
```

Comment:
It was changed when I added the "broken" option. Since @ekasper wants the tests to be in the new ssl test file, this part of the change is moot.